awscli/examples/eks/wait/nodegroup-deleted.rst

```diff
@@ -0,0 +1,9 @@
+**To wait for an nodegroup running in the Amazon EKS cluster to become deleted**
+
+The following ``wait nodegroup-deleted`` example command waits until `ResourceNotFoundException` is thrown when polling with `describe-nodegroup` for an nodegroup named ``my-nodegroup`` running in the Amazon EKS cluster named ``my-eks-cluster``. ::
```

Comment:
Same thing here, extra backticks for `ResourceNotFoundException` and `describe-nodegroup`.